README.md

```diff
@@ -4,3 +4,5 @@
 #Bene, Leon, Peter and Janos
 
 code, comments and conversation will be in english
+
+#We chose Contributor Covenant as our Code of Conduct
```

Comment:
A CoC is important but maybe not so important to get a level-1 header. ;) More common would be something like:

```
## Code of Conduct

This project adopted the [Contributor Covenant code of conduct](path/to/file/in/repo).
```